testing/conftest.py

```diff
@@ -51,9 +52,13 @@ def chrome_options(chrome_options):
         filterwarnings =
             error::DeprecationWarning
             ignore:--firefox-\w+ has been deprecated:DeprecationWarning
+            ignore:capabilities and desired_capabilities have been deprecated, please pass in a Service object:DeprecationWarning
+            ignore:firefox_profile has been deprecated, please use an Options object:DeprecationWarning
+            ignore:Setting a profile has been deprecated. Please use the set_preference and install_addons methods:DeprecationWarning
+            ignore:Getting a profile has been deprecated.:DeprecationWarning
```

Comment:
What's the effort of fixing this, so we don't get the warning to start with?

Also, we should clean out all the old deprecations (like `--firefox`), but that is probably a separate PR.

Reply:
I can have a look.

Reply:
Ok, thanks!

I've created a selenium-3 branch where we'll continue to support Selenium 3, while master will be the Selenium 4 branch going forward.

@dosas @isaulv Good? In that case we can merge this.

Reply:
(y)